Feature/adf 400/Tabular import - update internal documentation (#1776)
oat-sa/extension-tao-itemqti: model/import/README.md

Here you can find instructions about import items using TAO platform.

## Configuration

> The feature is disabled by default. To enable it, flag `FEATURE_FLAG_TABULAR_IMPORT_ENABLED` should be set to True at the env variables.

Reviewer: For the Score Setup, lets add a new option:

```
- *none*: In case there is only empty `choice_N_score` fields and no `correct_answer`.
```

And lets fix this one:

```
- *match_correct*: In case there are empty `choice_N_score` and `correct_answer` is provided.
```

> The feature is disabled by default. To enable it, flag `FEATURE_FLAG_TABULAR_IMPORT_ENABLED` should be set to True at the env variables.

Reviewer: ```suggestion
The feature is disabled by default. It can be enabled by the feature flasg `FEATURE_FLAG_TABULAR_IMPORT_ENABLED`.
```

Cause I think how to set feature flag is already described in another doc and we want to avoid duplicate this information.

### Examples

- You can see an [example here](./../../templates/import/import.sample.csv).

### Limitations

> In the current state of work, following are some limitations:
>    - Only choice interactions are supported for CSV 
>    - Text property type with one of the following types: Text Short Field, Text Long Box,  Text HTML editor values are supported

Reviewer: ```suggestion
- Only single/multiple choice interactions are supported.
- Only those types of metadata are supported: 
  - Text Short Field
  - Text Long Box
  - Text HTML editor
```

I think it is cleaner to read and also some grammar fixes.

>    - All metadata properties that define the “alias” attribute only considered

Reviewer: ```suggestion
   - Only metadata properties with the attribute “alias” will be considered.
```
